src/ixdat/spectra.py

```diff
@@ -296,6 +296,10 @@ def __add__(self, other):
             axes_series=[tseries, self.xseries],
         )
         spectrum_series_as_dict = self.as_dict()
+        technique = self.technique
```

Comment:
@ScottSoren I can't quite wrap my head around the database implications of changing the technique. Is it ok?

Reply:
I think there might be bad implications, haha...
But that's not what happens here! 

It is the technique for the _new object_ that gets returned which is being generated here.  That has to be recognized as a spectra technique rather than a spectrum technique, so that ixdat correctly identifies the result as a spectromeasurement when the spectra object (returned here) is added to a measurement.